Sales.java

```diff
@@ -0,0 +1,52 @@
+/*
+ * The Sales class keeps track of how many times an item has been ordered using
+ * a HashMap.
+ * */
+package model;
+
+import java.util.*;
+
+
+public class Sales {
+	/* I made sales HashMap String, Integer instead of food, integer because changing
+	 * using food as the key would require changing the hashcode, and equals method
+	 * in FoodData so contains is able to work. Possible to change if we absolutely need to though.
+	 * */
+	private HashMap<String, Integer> sales;
+	
+	// Constructor method
+	public Sales() {
+		sales = new HashMap<String, Integer>();
+	}
+	
+	
+	/* This method adds all the food in bill to the hashmap by checking the food name
+	 * and adding the quantity
+	 *  */
+	public void addCompletedOrder(Bill bill) {
+		for (FoodData food: bill.getOrder()) {
+			if(!(sales.keySet().contains(food.getName()))) {
+				sales.put(food.getName(), food.getQuantity());
+				continue;
+			}
+			sales.put(food.getName(), sales.get(food.getName()) + food.getQuantity());
+		}
+	}
```

Comment:
To ensure no unnecessary duplication, we should allow case-insensitive food name
```suggestion
			String foodName = food.getName().toLowerCase();
			if(!(sales.keySet().contains(foodName))) {
				sales.put(foodName, food.getQuantity());
				continue;
			}
			sales.put(foodName, sales.get(foodName) + food.getQuantity());
		}
	}
```